specs/core/0_beacon-chain.md

```diff
@@ -1789,7 +1793,7 @@ The steps below happen when `(state.slot + 1) % SLOTS_PER_EPOCH == 0`.
 
 * Let `previous_total_balance = get_total_balance(state, get_active_validator_indices(state.validator_registry, previous_epoch))`.
 * Validators that made an attestation during the previous epoch, targeting the previous justified slot:
-  * Let `previous_epoch_attestations = [a for a in state.latest_attestations if previous_epoch == slot_to_epoch(a.data.slot)]`. (Note: Each of these attestations votes for the previous justified epoch/block root because of the [attestation block validity rules](#attestations-1).)
+  * Let `previous_epoch_attestations = state.previous_epoch_attestations`.
```

Comment:
Maybe just get rid of this line and use `state.previous_epoch_attestations` everywhere? A 6 character reduction is not normally worth a helper variable.

Reply:
I left it as is because of the nice helper variable symmetry:

  * Let `previous_epoch_attestations`...
  * Let `previous_epoch_attester_indices`...
  * Let `previous_epoch_attesting_balance`...
  * Let `previous_epoch_boundary_attestations`...
  * Let `previous_epoch_boundary_attester_indices`...
  * Let `previous_epoch_boundary_attesting_balance`...
  * Let `previous_epoch_head_attestations`...
  * Let `previous_epoch_head_attester_indices`...
  * Let `previous_epoch_head_attesting_balance`...